roles/uname/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+
+- name: gather uname.os fact
+  raw: uname -s
+  register: uname_os
+  ignore_errors: yes
+
+- name: add uname.os to dictionary
```

Comment:
For Sonar we are flattening the fact structure. No need to add them to a dictionary